web/sidebars.js

```diff
@@ -119,6 +119,7 @@ module.exports = {
             'guides/database/extensions/uuid-ossp',
           ],
         },
+        'guides/database/resource-management',
```

Comment:
@burggraf / @kiwicopple - doesn't seem to be turning up on the preview at https://main-56wi8ry88-supabase.vercel.app/docs/guides ; any idea what I'm doing wrong?

Reply:
Shows up in https://docs-git-da-timeouts-supabase.vercel.app/docs/guides/database/resource-management